### PR TITLE
Small performance-related changes

### PR DESCRIPTION
**Pull request by nbougalis**

### CODE REVIEW

**Review by thejohnfreeman — APPROVED** [review]

File: src/ripple/app/misc/NetworkOPs.cpp [hunk 1]
```diff
@@ -4023,7 +4024,7 @@ NetworkOPsImp::getBookPage(
     Json::Value& jvOffers =
         (jvResult[jss::offers] = Json::Value(Json::arrayValue));
 
-    std::map<AccountID, STAmount> umBalance;
+    std::unordered_map<AccountID, STAmount> umBalance;
```
Comment (seelabs) [inline src/ripple/app/misc/NetworkOPs.cpp:4027]: May as well change the map in the disabled version of this function too.

**Review by seelabs — APPROVED** [review]
:+1:

Nit on the commit message: consider losing the phrase "If merged". I've seen other people do this as well. That commit message seems written to an audience of code reviewers before it is merged. I think it should be from the point of view of someone reading the logs after it's merged. Also, the "if merged" is implied.

**Review by pwang200 — APPROVED** [review]
:+1:

**Comment by MarkusTeufelberger** [comment]
`sweep_interval` is not documented in the default config or anywhere else than in the commit message.

**Comment by nbougalis** [comment]
Incorporated into #3981

**Pull request closed without merging**